content/en/docs/reference/setup-tools/kubeadm/kubeadm.md

```diff
@@ -15,6 +15,10 @@ kubeadm performs the actions necessary to get a minimum viable cluster up and ru
 
 Instead, we expect higher-level and more tailored tooling to be built on top of kubeadm, and ideally, using kubeadm as the basis of all deployments will make it easier to create conformant clusters.
 
+## How to install
+
+In common Linux you can install kubeadm by package manager, or use binary file for others: [Installing kubeadm](/docs/setup/production-environment/tools/kubeadm/install-kubeadm)
```

Comment:
i'd change this to:

```
To install kubeadm see the [installation guide](/docs/setup/production-environment/tools/kubeadm/install-kubeadm).
```

but will defer to sig-docs for the final say of how it's best to approach this.